ras_party/error_handlers.py

```diff
@@ -2,21 +2,37 @@
 
 import flask
 import structlog
-from flask import jsonify
+from flask import jsonify, request
 from requests import RequestException
 
-from ras_party.exceptions import RasError
+from ras_party.exceptions import ClientError, RasError
 
 logger = structlog.wrap_logger(logging.getLogger(__name__))
 
 blueprint = flask.Blueprint('error_handlers', __name__)
 
 
+@blueprint.app_errorhandler(ClientError)
+def client_error(error):
+    response = jsonify(error.to_dict())
+    response.status_code = error.status_code
+    logger.info('Client error',
+                url=request.url,
+                errors=error.to_dict(),
+                **error.kwargs,
+                status=error.status_code)
+    return response
+
+
 @blueprint.app_errorhandler(RasError)
 def ras_error(error):
     response = jsonify(error.to_dict())
     response.status_code = error.status_code
-    logger.exception('Uncaught exception', errors=error.to_dict(), status=error.status_code)
+    logger.exception('Uncaught exception',
+                     url=request.url,
+                     errors=error.to_dict(),
+                     **error.kwargs,
```

Comment:
Minor, but I think `**error.kwargs` should be the last argument, consistent with how it's used in function definitions.